mismi-sqs/src/Mismi/SQS/Aws/Commands.hs

```diff
@@ -90,18 +90,16 @@ toUserMessageAttributeValue v = do
     toScientific :: Text -> SQSAction Scientific
     toScientific = either fail pure . readEither . unpack
 
-toMessageAttribute :: Text -> SQSAction MessageAttribute
+toMessageAttribute :: A.MessageAttribute -> MessageAttribute
 toMessageAttribute = \case
-  "All" -> pure MessageAll
-  "ApproximateFirstReceiveTimestamp" -> pure ApproximateFirstReceiveTimestamp
-  "ApproximateReceiveCount" -> pure ApproximateReceiveCount
-  "SenderId" -> pure SenderId
-  "SentTimestamp" -> pure SenderId
-  u -> fail $ "Unsupported MessageAttribute: " <> unpack u
-
-toAttributes :: M.HashMap A.QueueAttributeName Text -> SQSAction [(MessageAttribute, T.Text)]
-toAttributes = traverse toPair . M.toList
-  where toPair (a, v) = (,) <$> toMessageAttribute (A.toText a) <*> pure v
+  A.All -> MessageAll
+  A.ApproximateFirstReceiveTimestamp -> ApproximateFirstReceiveTimestamp
+  A.ApproximateReceiveCount -> ApproximateReceiveCount
+  A.SenderId -> SenderId
+  A.SentTimestamp -> SenderId
```

Comment:
No, it is a mistake. (Un)fortunately this code is never run and will probably going to be removed soon. Will fix it meanwhile.